src/iuse_actor.cpp

```diff
@@ -2003,11 +2003,12 @@ long holster_actor::use( player *p, item *it, bool, const tripoint & ) const
 
 void holster_actor::info( const item&, std::vector<iteminfo>& dump ) const
 {
-    dump.emplace_back( "TOOL", _( "Can contain items up to " ), "<num> L",
-                       round_up( units::to_liter( max_volume ), 1 ), false, "", max_weight <= 0 );
+    int scale = 0;
+    dump.emplace_back( "TOOL", _( "Can contain items up to " ), string_format( "<num> %s", volume_units_abbr() ),
+                       round_up( convert_volume( max_volume.value(), &scale ), 1 ), scale == 0, "", max_weight <= 0 );
```

Comment:
Yes, but we don't have a `round_down` function yet. Could you tag that as an issue though and I'll write that function in a separate PR and also check for other suspicious calls to `round_up` in the codebase at the same time?